src/Browser.php

```diff
@@ -78,7 +78,7 @@ public function close()
      * @throws CommunicationException
      * @return Page
      */
-    public function createPage(): Page
+    public function createPage(array $options = []): Page
```

Comment:
**Useful Example**
`$browser->createPage(['preScript' => file_get_contents('do-not-detect.js')]);`

*do-not-detect.js*
```
// overwrite the `languages` property to use a custom getter
Object.defineProperty(navigator, 'languages', {
    get: function() {
        return ['en-US', 'en'];
    },
});

// overwrite the `plugins` property to use a custom getter
Object.defineProperty(navigator, 'plugins', {
    get: function() {
        // this just needs to have `length > 0`, but we could mock the plugins too
        return [1, 2, 3, 4, 5];
    },
});

// overwrite the `plugins` property to use a custom getter
Object.defineProperty(window, 'chrome', {
    get: function() {
        // this just needs to have `length > 0`, but we could mock the plugins too
        return [1, 2, 3, 4, 5];
    },
});

// set navigator.webdriver to false
Object.defineProperty(navigator, 'webdriver', {
    get: () => false,
});


// Simulate navigator permissions;
const originalQuery = window.navigator.permissions.query;
window.navigator.permissions.query = (parameters) => (
    parameters.name === 'notifications' ?
        Promise.resolve({ state: Notification.permission }) :
        originalQuery(parameters)
);
```